app/assets/stylesheets/custom.css.scss

```diff
@@ -152,3 +152,36 @@ p {
     color: #333;
   }
 }
+
+/* forms */
+
+input, textarea, select, .uneditable-input {
+  border: 1px solid #bbb;
```

Comment:
Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets<br>Each selector in a comma sequence should be on its own line